widget_driver/lib/src/drivable_widget.dart

```diff
@@ -99,7 +99,12 @@ class _DriverWidgetState<Driver extends WidgetDriver> extends State<DrivableWidg
   }
 
   bool _isRunningInTestEnvironment() {
-    return widget._environmentInfo.isRunningInTestEnvironment();
+    var isRunningInTest = false;
+    assert(() {
+      isRunningInTest = widget._environmentInfo.isRunningInTestEnvironment();
+      return true;
+    }());
+    return isRunningInTest;
```

Comment:
This is a style which the flutter team themselves use to remove parts of the code when running release builds.
All `assert` methods are removed from the compiles code when doing release builds.
So the code inside the assert only exists in debug.. for example during testing..

And for release, then the isRunningInTest will always be `var isRunningInTest = false;`
And then the dart-treeshaking comes into action and removes all code which is not accessible. Meaning all TestDriver creation